### PR TITLE
Added DAG file for Airflow Summit 2022

### PR DESCRIPTION
**Pull request by PeWu**
## Description

Added DAG file for Airflow Summit 2022.

The DAG will showcase autoscaling in Cloud Composer.

## Checklist
- [X] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [X] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [X] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [X] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by dandhlee** [comment]
Handing off to Leah, our Composer expert!

File: composer/2022_airflow_summit/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Airflow Summit 2022 sample DAGs
+
+## parallel_work
+
+Running a number of tasks in parallel showcases autoscaling in a Cloud Composer environment.
```
Comment (leahecole) [inline composer/2022_airflow_summit/README.md:5]:
```suggestion
# Airflow Summit 2022 Sample DAGs

DAGs in this directory are part of the curriculum for the [Cloud Composer Workshop](https://airflowsummit.org/sessions/2022/cloud-composer-workshop/) at the 2022 Airflow summit. Be sure to [register](https://ti.to/airflowsummit/2022-workshops) and join us there!  

## parallel_work

Running a number of tasks in parallel showcases autoscaling in a Cloud Composer environment.
```

**Review by leahecole — CHANGES_REQUESTED** [review]
Slight nit otherwise LGTM @PeWu !! Sorry for delay in review

**Review by leahecole — COMMENTED** [review]
Oh! Also! Please add a `requirements.txt` file that has Airflow in it and any other dependencies needed (I don't think you'll need any)
and a `requirements-test.txt` file that is identical to [this one](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/composer/workflows/requirements-test.txt)

File: composer/2022_airflow_summit/README.md (hunk 1 above)
Comment (PeWu) [reply composer/2022_airflow_summit/README.md]: Done

**Comment by PeWu** [comment]
> Oh! Also! Please add a `requirements.txt` file that has Airflow in it and any other dependencies needed (I don't think you'll need any) and a `requirements-test.txt` file that is identical to [this one](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/composer/workflows/requirements-test.txt)

Done

**Review by leahecole — APPROVED** [review]

**Review by parthea — APPROVED** [review]

**Pull request merged**